sklearn/cluster/tests/test_optics.py

```diff
@@ -217,236 +216,168 @@ def test_cluster_sigmin_pruning(reach, n_child, members):
 
 
 def test_reach_dists():
```

Comment:
This name is no longer valid, replace it with ``test_optics_ELKI`` or anything you like.